lib/cmds/utils.ts

```diff
@@ -112,10 +113,15 @@ export function convertArgs2Options(argv: yargs.Arguments): Options {
   if (argv.iedriver as boolean) {
     options.browserDrivers.push({name: 'iedriver', version: versionsIe});
   }
+  if (argv.gridNode as string === '') {
+    console.log('Please specify a grid hub URL...');
+    process.exit();
+  }
   if (argv.standalone as boolean) {
     options.server = {};
     options.server.name = 'selenium';
     options.server.runAsNode = argv.standalone_node as boolean;
+    options.server.gridNode = argv.gridNode as string;
```

Comment:
When grabbing things from the argv, we should use the `selenium_grid_url` and not the `standalone_` alias

Reply:
ok - done